site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,461 @@
+* How to create a Thread network on the OTBR.
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+## Prerequisites
+
+Duration: 3:00
+
+Hardware:
+
+1. 3 EFR32MGxx radio boards - you can use any combination of these devices. I used a BRD4166A as the RCP and two BRD4168As as Full Thread Devices.
+
+   * EFR32MG12 (BRD**4161A**, BRD**4166A**, BRD**4170A**, BRD**4304**A)
+   * EFR32MG13 (BRD**4168A**)
+   * EFR32MG21 (BRD**4180A**, BRD**4180B**)
+
+   If you are starting fresh, you can get one of the [EFR32 Thread starter kit](https://www.silabs.com/wireless/thread) which has the boards listed above.
+
+2. BRD4001A: Wireless starter main boards (WSTK) to host the radio boards. Except for BRD4166A, all the radio boards need a starter main board each. Mini USB cables to connect and power the main boards or micro USB cable for BRD4166A.
+
+> Note: Ensure the 3 way switch on your main board is in AEM position as shown below. The BRD4166A doesn't have this switch.
+
+![WSTK AEM](./img/WSTK_AEM.jpg)
+
+3. A Raspberry Pi 3B+ or greater with [Raspbian Stretch Lite OS image](http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip) or [Raspbian Stretch with Desktop](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip) which is connected to the internet over Ethernet. We configure this as an OT Border Router.
+
+4. A Windows/Linux/Mac Host system with at least 2 USB ports and an internet connection. Check the hardware and OS requirements at [SSv5](https://www.silabs.com/developers/simplicity-studio).
+
+5. At least one Ethernet cable for connecting the Raspberry Pi to Internet. The WSTKs also support debug and flashing over IP so, optionally, additional Ethernet cables can be used to connect the WSTK to your host system via an Ethernet Switch.
+
+Software:
+
+* [Simplicity Studio v5](https://www.silabs.com/developers/simplicity-studio) installed and updated on the Windows/Linux/Mac Host system with
+
+  * GNU ARM v7.2.1 toolchain 
+
+  * Gecko SDK Suite 3.1.0 or later and Silicon Labs OpenThread SDK.
+
+    > Note: The IAR Embedded Workbench for ARM (IAR EWARM) Compiler is currently not supported with OpenThread.
+
+
+
+## Hardware set up
+
+Duration: 3:00
+
+This Codelab was created using
+
+1. EFR32MG12 [BRD4166A Thunderboard Sense 2](https://www.silabs.com/development-tools/thunderboard/thunderboard-sense-two-kit) as shown on the left.
+2. Two EFR32MG13 [BRD4168A](https://www.silabs.com/documents/public/user-guides/ug280-brd4168a-user-guide.pdf) as shown on the right. 
+3. Simplicity Studio v5 installed on a macOS Catalina 10.15.7 with
+   1. Gecko SDK 3.1.2 
+   2. GNU ARM v7.2.1
+
+ ![](img/BRD4166A.jpeg)![BRD4168A](./img/BRD4168A.jpeg)
+
+> Note: You can update the GSDK or toolchain from **Studio -> Help -> Update Software -> SDKs/Toolchains**
```

Comment:
Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>